package.json

```diff
@@ -33,31 +33,40 @@
     "prelint": "cd samples; npm link ../; npm i"
   },
   "dependencies": {
+    "@octokit/rest": "^18.0.0",
+    "@types/node": "^14.0.13",
```

Comment:
I tend to add types as development dependencies, since they're not needed when you publish the library.